pkg/wallet/account.go

```diff
@@ -1,6 +1,6 @@
 package wallet
 
-import "github.com/anthdm/neo-go/pkg/util"
+import "github.com/CityOfZeon/neo-go/pkg/util"
```

Comment:
The CI job is failing as this is `Zeon` when it should be `Zion`, same with the other imports 👍

Reply:
Ah, thanks for that. Changed.